src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
```

Comment:
It is better not to shorten the names to unreadable ones (`cn` -> `classNames`).
This applies to all code.